src/xenia/gpu/gl4/premake5.lua

```diff
@@ -21,6 +21,7 @@ project("xenia-gpu-gl4")
   includedirs({
     project_root.."/third_party/gflags/src",
   })
+  filter("platforms:Windows")
```

Comment:
I'll look if I can make gl4 Windows only from the main premake file, similar to xaudio2 and friends.